MoreLinq/NoConflict.ToDataTable.cs

```diff
@@ -0,0 +1,89 @@
+namespace MoreLinq.NoConflict
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq.Expressions;
+
+    /// <summary><c>ToDataTable</c> extension.</summary>
+    public static partial class ToDataTableExtension
+    {
+        /// <summary>
+        /// Appends elements in the sequence as rows of a given <see cref = "DataTable"/> 
```

Comment:
All the xmldocs appear to have spaces around the attribute assignments. I have no idea if this has any impact though, at least the Visual Studio 2015 tooltips seem to have no problem with it.